src/commands/deploy.rs

```diff
@@ -384,17 +384,21 @@ impl DeployCommand {
                 existing_channel_id
             }
             Err(_) => {
-                let range_rule = Some(bindle_id.version_string());
                 let app_id = CloudClient::add_app(&client, &name, &name)
                     .await
                     .context("Unable to create app")?;
+
+                let active_revision_id = self
```

Comment:
We don't seem to be calling `CloudClient::add_revision` on this path (as we do on the "app exists" path.  Do we understand why it is unnecessary on this path but necessary on the other?  Could we maybe add a comment for the benefit of a maintainer who comes across this and is puzzled like me?  Thanks!

Reply:
https://github.com/deislabs/hippo/blob/e83bda6858c5d5b8db234f6f07bbf4b1afb5f8d2/src/Application/Apps/EventHandlers/InitialRevisionImport.cs#L9  - on app creation, we import all existing revisions into db. therefore we don't need add_revision to be called explicitly (I did added that initially, but it was creating a duplicate revision entry into db).

added this as a comment in code as well. 

thank you for pointing it out